.gitignore

```diff
@@ -38,3 +38,8 @@ githooks/*.dll
 
 # VS poops
```

Comment:
could you remove "VS poops" while you're in here?

Reply:
Done, thanks for catching that.